ext/drreg/drreg.c

```diff
@@ -68,14 +69,27 @@
 /* This should be pretty hard to exceed as there aren't this many GPRs */
 #define MAX_SPILLS (SPILL_SLOT_MAX + 8)
 
-#define AFLAGS_SLOT 0 /* always */
+/* This should be pretty hard to exceed as there aren't this many SIMD registers */
```

Comment:
I agree with you. I was confused too. Perhaps it was some form of non-standard defensive programming to prevent some out of bounds error. Maybe there was/is a difficult bug that was too hard to find and this was a quick solution ???

Reply:
Ah, I think I understood why we have additional spill slots. Its mainly for spilling client values.

Reply:
I see now. It means that there are enough slots of all GPRs _and_ client spills. Feel free to clarify this. Especially this isn't true or applicable to the new line you've added, since this is only used for the new indirect tls space.

Reply:
Done.